build.gradle.kts

```
@@ -44,6 +44,9 @@ dependencies {
 
     developmentOnly("org.springframework.boot:spring-boot-devtools")
     testImplementation("org.springframework.boot:spring-boot-starter-test")
+
+    implementation("org.zalando:problem-spring-web-starter:0.27.0")
+    implementation("org.springframework.boot:spring-boot-starter-validation")
```

```suggestion
    implementation("org.springframework.boot:spring-boot-starter-actuator")
    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
    implementation("org.springframework.boot:spring-boot-starter-web")
    implementation("org.springframework.boot:spring-boot-starter-validation")
```
다음과 같이 spring-starter끼리 모아보는것은 어떨까요?

---

나름 생각해서 구분 해 보았어요